cryptomatte/cryptomatte.h

```diff
@@ -567,7 +567,7 @@ inline void write_manifest_to_string(const ManifestMap& map, String& manf_string
         uint32_t float_bits;
         std::memcpy(&float_bits, &hash_value, 4);
         char hex_chars[9];
-        sprintf(hex_chars, "%08x", float_bits);
+        snprintf(hex_chars, 9, "%08x", float_bits);
```

Comment:
Use sizeof.  Same will apply to all the other snprintf usage, so I won't mention each case.